packages/mattermost-redux/src/client/client4.ts

```diff
@@ -2548,6 +2548,20 @@ export default class Client4 {
         );
     };
 
+    getAppsOAuthAppIDs = () => {
```

Comment:
Is there any chance either of these will ever need paging in the future? That's obviously a bigger question than just something that affects the web app, but I find it never hurts to ask

Reply:
Being small amount of information (a list of ids) I hope there is no case where this may become a problem (e.g. having 100 Apps installed?). That being said, I am open to suggestions or comments.